### DISCUSSION TITLE
Format timezone facts tag template

### DISCUSSION THREAD

**Discussion by xcloudx87** [argoproj-labs/argocd-notifications#311] [Q&A]
Hi there,

I am creating template for notification and i want to format timezone but this template didn't work.

> facts: |
        [
        {
          "name": "Author",
          "value": "{{(call .repo.GetCommitMetadata .app.status.sync.revision).Author}}"
        },
        {
          "name": "Commit Message",
          "value": "{{(call .repo.GetCommitMetadata .app.status.sync.revision).Message}}"
        },
        {
          "name": "Sync Status",
          "value": "{{.app.status.sync.status}}"
        },
        {
          "name": "Synced at",
          "value": "{{ (call .time.Parse .app.status.operationState.finishedAt).Local.Format "2006-01-02T15:04:05Z07:00" }}"
        },
        {
          "name": "Repository",
          "value": "{{.app.spec.source.repoURL}}"
        }
        ]

It still shows time in UTC timezone. Do you guy know how to use function with double quote in facts tag?

Thanks in advance!

**Comment by ryota-sakamoto** [comment]
Hi @xcloudx87 
Have you defined `TZ` in `env`?
https://argocd-notifications.readthedocs.io/en/stable/templates/#change-the-timezone

**Reply by xcloudx87** [reply]
Hi @ryota-sakamoto,

Thank you for your answer.

Yes, I have defined TZ in env. But the result return is still in UTC timezone. Please look at below for reference:

![image](https://user-images.githubusercontent.com/71874570/125251094-25a61780-e321-11eb-8e32-b5b0b1e7c019.png)


The log which returns from pod's console shows correct timezone but "app.status.operationState.finishedAt" field still returns in UTC timezone.

BTW I am using argocd-notifications with latest version (1.1.1).

Please advise. 

Thanks in advance!
